content/tests/components/recipes/details/RecipeDetails.test.js

```diff
@@ -141,6 +162,30 @@ describe("The `RecipeDetailForm` component", () => {
     }
   });
 
+  it("displays details from experimenter", async () => {
+    const recipeData = branchedAddonSetup();
+    setup(recipeData);
+    const { getByText, findByText, findByTestId } = await render(<App />);
+    fireEvent.click(getByText("Recipes"));
+    fireEvent.click(await findByText(recipeData.latest_revision.name));
+
+    await waitFor(() =>
+      expect(ExperimenterAPI.prototype.fetchExperiment).toReturn(),
+    );
+    await waitFor(() => expect(NormandyAPI.prototype.fetchRecipe).toReturn());
+
+    expect(getByText("Experimenter Details")).toBeInTheDocument();
```

Comment:
I find it hard to be sure that this test is right since the experimenter definitions are pretty far away. Just reading this test it isn't clear that the proposed start date should be "Sep 08", for example.

I think that following the example of how recipe data is handled with `setup` might be easier: Then the experimenter definitions that matter can be defined in this test with a factory, while still abstracting out all the API mocking.